tests/misc_env/install_prereq.py

```diff
@@ -115,6 +115,12 @@ def run(**kw):
                     workaround_openssl_issue,
                     ceph,
                 )
+
+    # enable coredump on all nodes
+    with parallel() as p:
+        for ceph in ceph_nodes:
+            p.spawn(enable_coredump, ceph)
+            time.sleep(20)
```

Comment:
Could we add it to `def install_prereq(...)` rather running through the thread again?